acrobat/blocks/app-banner/app-banner.js

```diff
@@ -0,0 +1,130 @@
+import { setLibs } from '../../scripts/utils.js';
+
+const miloLibs = setLibs('/libs');
+/**
+ * Reusing from express codebase,
+ * can be replaced once Milo provides an implementation
+ * @returns mobileOS
+ */
+function getMobileOperatingSystem() {
+  const userAgent = navigator.userAgent || navigator.vendor || window.opera;
+
+  // Windows Phone must come first because its UA also contains "Android"
+  if (/windows phone/i.test(userAgent)) {
+    return 'Windows Phone';
+  }
```

Comment:
Do we handle Windows Phone specially, or just let it go to "unknown"?

Reply:
not sure if anyone uses Windows Phone, but keeping it anyway.